base/src/main/proto/spine/options.proto

```diff
@@ -795,6 +795,33 @@ message MinOption {
     //     uint32 size = 2 [(min).value = "-5"]; // Falls below the `uint32` minimum.
     // }
     //
+    // ## Field references
+    //
+    // Instead of numeric literals, you can reference another numeric field.
+    // At runtime, the field’s value will be used as the bound. Nested fields are supported.
+    //
+    // Example: defining minimum values using field references.
+    //
+    // message MinimumValues {
+    //
+    //     int32 min_employees = 1;
+    //     int32 employees = 2 [(min).value = "min_employees"];
```

Comment:
`min_employees` is not a really good example. Can we be in the "range" of temperatures in various scales, ranges, etc. That is, without touching sensitive matters like people. Please.